### PR TITLE
Improved pre-requisites

### PR DESCRIPTION
**Pull request by ramonsmits**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,12 +10,34 @@ If you have any difficulty preparing your machine, or following this document, p
 - [Demos](#demos)
 - [FAQ](#faq)
 
-## Preparing your machine for the workshop
+## Preparing for the workshop
```
Comment (adamralph) [inline README.md]: I'd prefer to leave this section title as-is.

File: README.md [hunk 2]
```diff
@@ -10,12 +10,34 @@ If you have any difficulty preparing your machine, or following this document, p
...
+  - [Visual Studio 2017 v15.9.8+](#visual-studio)
+    - .NET desktop development
+    - ASP.NET and web development
+    - .NET Core SDK 2.0.3+
+  - Setup [SQL Server 2017 LocalDB](#localdb) (15 min)
+  - Setup [SQLCMD](#sqlcmd) (15 min)
+- [Get a copy of this repository](#get-a-copy-of-this-repository) (10 min)
+- [Set up the databases](#set-up-the-databases) (5 min)
+- Verify correct installation of pre-requisites (5 min)
+  - [Build the exercise solutions](#build-the-exercise-solutions)
```
Comment (adamralph) [inline README.md]: I'd prefer not to add this list. It duplicates what's below which means more maintenance burden, and I'm not sure the granular timings add much value.

File: README.md [hunk 3]
```diff
@@ -10,12 +10,34 @@ If you have any difficulty preparing your machine, or following this document, p
...
+- Verify correct installation of pre-requisites (5 min)
+  - [Build the exercise solutions](#build-the-exercise-solutions) 
+  
+
+You can also [prepare your machine in advance for the advanced exercises](#preparing-your-machine-for-the-advanced-exercises) but it is not a requirement for the workshop.
+
+- Setup Particular Platform Installer
+  - [Run the Particular Platform Installer](#run-the-particular-platform-installer)
+  - [Add ServiceControl Instance](#add-servicecontrol-instance)
+  - [Add Monitoring Instance](#add-monitoring-instance)
```
Comment (adamralph) [inline README.md]: Similarly, I'd prefer not to add this list here.

File: README.md [hunk 4]
```diff
@@ -10,12 +10,34 @@ If you have any difficulty preparing your machine, or following this document, p
...
 
-- [Install the pre-requisites](#install-the-pre-requisites)
-- [Get a copy of this repository](#get-a-copy-of-this-repository)
-- [Set up the databases](#set-up-the-databases)
-- [Build the exercise solutions](#build-the-exercise-solutions)
```
Comment (adamralph) [inline README.md]: I'd prefer to leave this in place.

File: README.md [hunk 5]
```diff
@@ -118,12 +140,16 @@ You can now attempt the advanced exercises. Don't worry if you don't manage to s
 
 It is likely that you will have to read documentation for the Particular Service Platform to finish the advanced exercises. Links to documentation will be provided.
 
+Note: When using Microsoft Edge please read [How do I download the Particular Platform Installer with Microsoft Edge?](#how-do-i-download-the-particular-platform-installer-with-microsoft-edge)
```
Comment (adamralph) [inline README.md]: I'd prefer to use the same note that is used in the standard exercise prep:

```suggestion
NOTE: If you are using Microsoft Edge, see [the FAQ](#how-do-i-download-the-particular-platform-installer-with-microsoft-edge).
```

File: README.md [hunk 6]
```diff
@@ -132,7 +158,7 @@ When installation has completed, click "Start ServiceControl Management".
 
 You may now close the Particular Service Platform Installation window.
 
-### Configure ServiceControl Instance
+### Add ServiceControl Instance
```
Comment (adamralph) [inline README.md]:
```suggestion
### Add a ServiceControl Instance
```

File: README.md [hunk 7]
```diff
@@ -147,7 +173,7 @@ Click "Add".
 
 After the instance has been added, make a note of the name of the instance. This will usually be "Particular.ServiceControl".
 
-### Configure Monitoring Instance
+### Add Monitoring Instance
```
Comment (adamralph) [inline README.md]:
```suggestion
### Add a ServiceControl monitoring Instance
```

**Comment by dvdstelt** [comment]
Sorry, but I agree with every comment Adam made. Not sure what the changes that you're suggesting, add to the current document?

**Comment by ramonsmits** [comment]
The document does not contain a easy list of things to do. It currently is a wall of text where the reader is forced to read which is not needed. A list of things to do as an overview is convenient.

**Comment by ramonsmits** [comment]
As there it is deemed not useful I'll close the PR and just email this list instead.

**Comment by ramonsmits** [comment]
For reference the text I communicated to the customer:

Please communicate to the attendees to have their environment prepared before the workshop.

**Preparation time can be up to 2 hours!**


- Setup Windows 10 (virtual) machine (20 min)
  * Enabled features: MSMQ and MSDTC
- Visual Studio 2017 v15.9.8+ (15 min)
  * .NET desktop development
  * ASP.NET and web development
  * .NET Core SDK 2.0.3+
- SQL Server 2017 LocalDB (15 min)
- Setup SQLCMD (15 min)
- Cloning the workshop repository (git clone or zip download)
- Setup required databases (5 min)
- Verify correct installation of pre-requisites (5 min)

Detailed instructions can be found at :  https://github.com/Particular/Workshop/blob/master/README.md

You can also prepare for the advanced exercises but it is not a requirement for the workshop.

- Download and Run the Particular Platform Installer
- Add a ServiceControl Instance
- Add a Monitoring Instance

**Pull request closed without merging**